packages/docusaurus-plugin-content-blog/src/__tests__/index.test.ts

```diff
@@ -80,20 +84,28 @@ describe('loadBlog', () => {
       truncated: false,
     });
 
-    expect(
-      blogPosts.find(v => v.metadata.title === 'no date').metadata,
-    ).toEqual({
+    expect({
+      ...blogPosts.find(v => v.metadata.title === 'no date').metadata,
+      ...{prevItem: undefined},
```

Comment:
It was necessary to modify the existing tests here and below, so that the tests pass successfully in the **CI**.

The fact is that in fixtures there is a `no date` file, it does not have a date, which means that it is calculated from the _creation_ _date_. But the “oddity” of CI is that it copies, as I suppose this file, and therefore its creation date will always be equal to the _current test execution time_, which is why expected and received data no longer match!

As I wrote above, this issue applies only to CI, locally tests pass successfully without these changes. But in order to achieve successful execution of tests in CI, we need to exclude the `prevItem` key from the `metadata` object since this element will change in the **CI environment** (CIrcieCI).

I don’t know how clearly I have stated the essence of the problem, but in this case I consider such a solution acceptable, because we ran into details of handling this CI.